source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -260,23 +245,17 @@ This product's shipping availability as used throughout your shop.
 <%= settings.site_name.capitalize %> uses a fixed set of availability
 descriptions that will be crosslinked to the ones provided in your feed.
 
-Available
-: refers to products that are available for pick up at your outlet
-
-1 to 3 days
-: refers to products that can be delivered to your customer within 1 to 3 days
+Available - Instant Purchase / Delivery 1 to 3 days
```

Comment:
Changed this line:

```
Available - Instant Purchase / Delivery 1 to 3 days
```

to

```
Available in store / Delivery 1 to 3 days
```

in 21925e876e16dd08544f7010e69a500bb03266ae.